frontend/styles/redeem/indexStyles.ts

```diff
@@ -0,0 +1,7 @@
+import { SxProps } from '@mui/material';
+
+export const containerSx: SxProps = {
+  maxWidth: '1200px',
```

Comment:
How was this picked?

Reply:
https://github.com/Giving-Coupons/giving-coupons/pull/66#discussion_r990742075